Makefile

```diff
@@ -259,3 +261,57 @@ datasets/enwiki.draft_quality.201608-201701.with_cache.json.bz2: \
 	articlequality extract_from_text \
 	  draftquality.feature_lists.enwiki.draft_quality \
 	  --verbose | bzip2 -c > $@
+
+#################### Portuguese Wikipedia #######################
+datasets/ptwiki.draft_quality.201903202003.json:
+	wget -q https://quarry.wmflabs.org/run/444292/output/0/json-lines?download=true > $@
```

Comment:
Change this to: 
`wget -qO- https://quarry.wmflabs.org/run/444292/output/0/json-lines?download=true > $@`